actions/cwrhelpers.py

```diff
@@ -118,3 +119,69 @@ def wait_result(jclient, job_name, build_number, secs_to_wait=60):
             print("Jenkins job {} not running yet".format(build_number))
         except:
             raise
+
+
+def get_s3_credentials(cred_name=None):
+    """Get S3 credentials from the juju credentials command.
+
+    if cred_name is set, get the credentials for the name.
+    if cred_name is not set, try to get the credentials for the default name.
+    if cred_name is not set, no default credential name and there is a single
+      credential name, get the credentials for that name.
+
+    Returns: access_key, secret_key
+    """
+    cmd = 'juju credentials aws --format yaml --show-secrets'.split()
+    try:
+        creds = subprocess.check_output(cmd)
+    except subprocess.CalledProcessError as e:
+        fail_action(
+            "Error running 'juju credentials' command: ".format(e.output))
+        return None
+    creds = yaml.load(creds)
+    if not creds.get('credentials', {}).get('aws'):
+        fail_action('AWS credentials not found. Set AWS credentials by '
+                    'running "set-credentials" action.')
+        return None
+    if not cred_name:
+        cred_name = creds.get('credentials', {}).get('aws', {}).get(
+            'default-credential')
+    if (not cred_name and
+            len(creds.get('credentials', {}).get('aws', {}).keys()) == 1):
+        cred_name = list(creds.get('credentials', {}).get('aws', {}).keys())[0]
+    if not cred_name:
+        fail_action('Credentials not found. Set AWS credentials by '
+                    'running "set-credentials" action.')
+        return None
```

Comment:
It was added for the unit tests. Since the `fail_action` is mocked for having `sys.exit`, not having `return` produces incorrect results  when the unit test continues executing the rest of the code.